index.js

```diff
@@ -197,6 +197,18 @@ function getPattern(name) {
   )
 }
 
+
+/**
+ * A value of cookie can be quoted-string (cf. RFC 2965) and in that case, stripping the quotes is useful for applications.
+ */
+
+function stripQuotes(value) {
```

Comment:
Note that the logic here should be [the same as another cookie library](https://github.com/jshttp/cookie/blob/0b519534a5d0bea176f8422aeb93f7d9fce8d683/index.js#L72), which is used in Express.